Firestore/Source/Util/FSTLogger.m

```diff
@@ -16,7 +16,12 @@
 
 #import "Firestore/Source/Util/FSTLogger.h"
 
-#import "FIRLogger.h"
+#ifdef COCOAPODS
```

Comment:
These get written by copybara when you import a well known header.

In the open source repo, this should just be `#import <FirebaseCore/FIRLogger.h>`.